### ISSUE TITLE
Upload `.cls` to CTAN

### ISSUE THREAD

**Issue by roaldarbol** [roaldarbol/LaPreprint#14]
Consider uploading the `.cls` to CTAN so the document class can be included in TeXLive and MiKTeX, and the template doesn't need to include the `.cls` file.

**Comment by hvoss49** [comment]
The template can be part of the documentation. Your upload should have the following structure:

```
doc/     ->  everything without the .cls
latex/    -> the .cls file
README     
Changes  -> short log for updates
```

The Changes file can look like

```
0.02  2022-11-10      - bugfix for ....
0.01   2022-10-30    - first CTAN version

```

**Comment by roaldarbol** [comment]
@hvoss49 I'm looking into this currently. I read a lot about `.dtx` and `.ini` files. For this, do I need to convert the `.cls` file into `.dtx` or would it suffice to just zip a folder of the structure you mentioned above?

**Comment by hvoss49** [comment]
No, you can upload it as a zip file with the above directory structure. Creating a  .dtx and .ini file makes only sense, if you are interested in further working with LaTeX.

**Comment by roaldarbol** [comment]
Perfect, thanks for the clarification!
